### PR TITLE
docs: add container architecture

### PR DESCRIPTION
**Pull request by SuperCipher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,8 +132,20 @@ how to operate in advance use cases, use case by use case
...
 
 # Architecture
+![Container architecture](/docs/assets/container-architecture.jpg "Container architecture")
 
-deeper details of how things work together
+All these component work together in stack means that if the lower part is missing then some of the API in [rest.bitcoin.com](https://github.com/ActorForth/rest.bitcoin.com) might not function properly.
+
+The main reason that you should use all of our fork stack is because we maintain and guarantee that all of the components will work well with the Regtest address format. Allow you to test locally more easily.
+
+The blue part is the node that you choose. Default node is Bitcoin unlimited. Currently this node is always required.
```
Comment (merc1er) [inline README.md:141]: Isn't Bitcoin Cash Node also an option?

File: README.md [hunk 2]
```diff
@@ -132,8 +132,20 @@ how to operate in advance use cases, use case by use case
...
+
+The red part is an indexer. ElectrsCash is for indexing Bitcoin cash transaction and aggregate BCH balance per address. Without these you won't be able to query the amount of BCH you own in a particular address. OpenSight is a shim micro services to make ElectrsCash compatible with [rest.bitcoin.com](https://github.com/ActorForth/rest.bitcoin.com) API interface.
+SLPDB is an indexer for SLP token, color coin on top of Bitcoin Cash, this will verified if the data relate to the color coin is valid and checking if the coin follow the SLP off-chain concensus.
```
Comment (merc1er) [inline README.md:144]: Maybe link to https://slp.dev/tooling/slpdb/#what-is-slpdb here

**Review by merc1er — COMMENTED** [review]
I have some feedback on the rest of the README - will send this to you directly @SuperCipher

File: README.md (hunk 1 above)
Comment (SuperCipher) [reply README.md:141]: Right. I will add that

**Pull request merged**